hhs_oauth_server/settings/base.py

```diff
@@ -78,6 +80,12 @@
 
 ]
 
+REST_FRAMEWORK = {
+    'DEFAULT_THROTTLE_RATES': {
+        'token': None,
```

Comment:
Are we actually sending the headers if the limit is set to None? What are users actually returning on a request?

Reply:
If the limit is set to `None` no headers will be set (as their values wouldn't make sense). When the limit is set, to something like `10000/hour`, the response will have something like
```
X-RateLimit-Limit: 10000
X-RateLimit-Remaining: 9999
X-RateLimit-Reset: 1372700873
```

Reply:
So, we want to make sure we actually return the headers in some form, just so that users can actually see what they will look like and develop against them as if they're functional. This could just mean setting them to something absurdly high - like a million per minute - and it could also mean faking them. But we do actually need to return the headers on live requests.

Reply:
OK I've made the setting an env var so it can be set to somthing crazy high, like `1000000/s`.